cloud/aws/modules/secrets_manager/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "app_secret_key_version" {
```

Comment:
nit: please pick either including "version" or not in the name for consistency. (I prefer not)